### PR TITLE
Feature/hearing schedule and assign judge pages components and navigation - and integration tests

### PR DESCRIPTION
**Pull request by dessap**
### Change description ###
Admin Web hearing schedule and assign judge pages components and navigation.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: AcceptanceTests/AcceptanceTests.PageObject/Helpers/PageNavigator.cs [hunk 1]
```diff
@@ -11,39 +12,53 @@ public class PageNavigator
...
                         CurrentPage.Continue();
-                    }
-                    catch (Exception ex)
-                    {
-                        Console.WriteLine($"Exception occurred {ex.Message}");
-                        Console.WriteLine("Page has no forms to be filled.");
-                        Console.WriteLine("Continuing to next page.");
-                        CurrentPage.Continue();
-                    }
+                    //}
```
Comment (npoxon) [inline AcceptanceTests/AcceptanceTests.PageObject/Helpers/PageNavigator.cs]: Is this commented out code still needed?

File: AcceptanceTests/AcceptanceTests.PageObject/Pages/AdminWebsite/AddParticipantsPage.cs [hunk 2]
```diff
@@ -1,24 +1,33 @@
...
+            };
         }
 
-        public void FillInAddress(Address address)
-        {
-            _addressForm = new AddressForm(WrappedDriver);
-            _addressForm.AssertComponentLoaded(this);
-            _addressForm.FillFormDetails(address);
-        }
+        //public void FillInAddress(Address address)
```
Comment (npoxon) [inline AcceptanceTests/AcceptanceTests.PageObject/Pages/AdminWebsite/AddParticipantsPage.cs]: ..and this?

File: AcceptanceTests/AcceptanceTests.Model/FormData/AddParticipantDatails.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        public string DisplayName { get; set; }
+        public IFormData Address { get; set; }
+
+        public IFormData GenerateFake()
+        {
+            Party = new DropdownListFormData();
+            Role = new DropdownListFormData();
+            FirstName = Faker.Name.First();
+            LastName = Faker.Name.Last();
+            Email = $"{FirstName}.{LastName}@{Faker.Internet.DomainName()}";
```
Comment (npoxon) [inline AcceptanceTests/AcceptanceTests.Model/FormData/AddParticipantDatails.cs]: can you append this with "Automation_"? Same with all the fake emails and first names?

Comment (dessap) [reply AcceptanceTests/AcceptanceTests.Model/FormData/AddParticipantDatails.cs]: done.

File: AcceptanceTests/AcceptanceTests.PageObject/Helpers/PageNavigator.cs (hunk 1 above)
Comment (dessap) [reply AcceptanceTests/AcceptanceTests.PageObject/Helpers/PageNavigator.cs]: Removed.

File: AcceptanceTests/AcceptanceTests.PageObject/Pages/AdminWebsite/AddParticipantsPage.cs (hunk 2 above)
Comment (dessap) [reply AcceptanceTests/AcceptanceTests.PageObject/Pages/AdminWebsite/AddParticipantsPage.cs]: Testing

**Pull request merged**